htdocs/product/stock/massstockmove.php

```diff
@@ -336,11 +336,11 @@
 					continue;
 				}
 				//var_dump($data);
-				$tmp_id_sw = $data[$i][0]['val'];
-				$tmp_id_tw = $data[$i][1]['val'];
-				$tmp_id_product = $data[$i][2]['val'];
-				$tmp_qty = $data[$i][3]['val'];
-				$tmp_batch = $data[$i][4]['val'];
+				$tmp_id_sw = dol_escape_htmltag($data[$i][0]['val']);
+				$tmp_id_tw = dol_escape_htmltag($data[$i][1]['val']);
+				$tmp_id_product = dol_escape_htmltag($data[$i][2]['val']);
```

Comment:
dol_escape_htmltag is used to sanitize string. When we are sure data is an int, we must use (int) or (float)
So tmp_id_product =  (int) $data[$i][2]['val']